### PR TITLE
Added dash-simple-image-processor app

### PR DESCRIPTION
**Pull request by nicholas-esterer**
# App pull request
- This is a new app

## About

- Playground deployment URL [playground url]/dash-simple-image-processor
- Current gallery app URL:  [gallery url]/dash-simple-image-processor

## Workflow

- I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [ ] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.

## The pre-review review

I have addressed all of the following questions:

- I have removed
  any dead and/or irrelevant code.
- My code is
  readable and, where it isn't, it has been commented appropriately.
- I have used appropriate packages to
  lessen the volume of code that needs to be maintained.

### CODE REVIEW

File: apps/dash-simple-image-processor/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+black==19.10b0
+dash-canvas==0.1.0
```
Comment (emmanuelle) [inline apps/dash-simple-image-processor/requirements.txt:2]: is dash-canvas used anywhere in this demo? I also don't think black is required here, not sure about wheel.

Comment (emmanuelle) [reply apps/dash-simple-image-processor/requirements.txt:2]: you need scikit-image though for your `io_utils.py`, and scikit-image is a dependency of `dash-canvas`. But if `dash-canvas` is not needed per se it's better to just put scikit-image in the requirements.

File: apps/dash-simple-image-processor/assets/swansea/SWANSE__-392e.txt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+PAYMENT FOR USE
+
+Provided this font is used for personal purposes then no fee is payable to 
+the author for it's use. You may use this font yourself and may pass on 
+copies to your friends provided that the copy contains all of the files 
+from the original package. Distribution can be by electronic means or on 
+floppy disk.
+
+If this font is used for commercial purposes then the user agrees to make a 
+one off payment to the author of �25.00.
```
Comment (emmanuelle) [inline apps/dash-simple-image-processor/assets/swansea/SWANSE__-392e.txt:37]: I think it's a problem to include files with such a license in this repo. What do you think @nicholas-esterer @VeraZab ?

**Comment by emmanuelle** [comment]
Thanks @nicholas-esterer ! My usual rant about making `requirements.txt` as human-readable as possible ;-). Otherwise my main concern is about the license of the fonts, I'm not sure we can use them here (without paying the 25$ mentioned in the license).

**Comment by nicholas-esterer** [comment]
Ha, didn't notice that in the licence file. Yes I'll change the font haha.

File: apps/dash-simple-image-processor/requirements.txt (hunk 1 above)
Comment (nicholas-esterer) [reply apps/dash-simple-image-processor/requirements.txt:2]: This is the output of `pipdeptree | grep -P '^\w+'` which programmatically determines the minimum `requirements.txt` file. But it's true `black` is probably not needed.

Comment (nicholas-esterer) [reply apps/dash-simple-image-processor/requirements.txt:2]: Basically `dash-canvas` requires everything we need haha.

Comment (nicholas-esterer) [reply apps/dash-simple-image-processor/requirements.txt:2]: `scikit-image` is installed as a dependency of `dash-canvas`
```
dash-canvas==0.1.0
  - dash [required: >=1.6.1, installed: 1.12.0]
    - dash-core-components [required: ==1.10.0, installed: 1.10.0]
    - dash-html-components [required: ==1.0.3, installed: 1.0.3]
    - dash-renderer [required: ==1.4.1, installed: 1.4.1]
    - dash-table [required: ==4.7.0, installed: 4.7.0]
    - Flask [required: >=1.0.2, installed: 1.1.2]
      - click [required: >=5.1, installed: 7.1.2]
      - itsdangerous [required: >=0.24, installed: 1.1.0]
      - Jinja2 [required: >=2.10.1, installed: 2.11.2]
        - MarkupSafe [required: >=0.23, installed: 1.1.1]
      - Werkzeug [required: >=0.15, installed: 1.0.1]
    - flask-compress [required: Any, installed: 1.5.0]
      - brotli [required: Any, installed: 1.0.7]
      - flask [required: Any, installed: 1.1.2]
        - click [required: >=5.1, installed: 7.1.2]
        - itsdangerous [required: >=0.24, installed: 1.1.0]
        - Jinja2 [required: >=2.10.1, installed: 2.11.2]
          - MarkupSafe [required: >=0.23, installed: 1.1.1]
        - Werkzeug [required: >=0.15, installed: 1.0.1]
    - future [required: Any, installed: 0.18.2]
    - plotly [required: Any, installed: 4.8.1]
      - retrying [required: >=1.3.3, installed: 1.3.3]
        - six [required: >=1.7.0, installed: 1.15.0]
      - six [required: Any, installed: 1.15.0]
  - Pillow [required: Any, installed: 7.1.2]
  - scikit-image [required: Any, installed: 0.17.2]
```

Comment (nicholas-esterer) [reply apps/dash-simple-image-processor/requirements.txt:2]: Basically what I'm trying to get at is that  the best looking `requirements.txt` is not really well defined ;). But I understand we're supposed to showcase `dash` and `scikit-image`, so what if I just include those 2 as well, so people that look won't be confused (it shouldn't break anything).

**Comment by xhlulu** [comment]
We are currently cleaning this repository using BFG, which requires all open PRs to be closed. Feel free to recreate a new PR with the same commits after the repository is cleaned.

**Pull request closed without merging**